platform/nservicebus.include.md

```diff
@@ -59,7 +59,7 @@ Message handlers and sagas can be [tested in isolation](/nservicebus/testing/).
 
 ## Flexible
 
-NServiceBus endpoints can be hosted anywhere code can be executed, such as in a Windows Service, a Docker container, or in the cloud with Azure or AWS. NServiceBus is compiled against .NET Standard 2.0 and endpoints can be run on [a variety of platforms](/nservicebus/upgrades/supported-platforms.md).  
+NServiceBus endpoints can be hosted anywhere code can be executed, such as in a Windows Service, a Docker container, or in the cloud with Azure or AWS. NServiceBus is compiled against .Net Framework 4.7.2 and .NET Core 2.1 allowing endpoints to run on [a variety of platforms](/nservicebus/upgrades/supported-platforms.md).  
```

Comment:
I'd prefer if we could not mention specific framework targets here. Seems like something that will just get overlooked and outdated again.

```suggestion
NServiceBus endpoints can be hosted anywhere code can be executed, such as in a Windows Service, a Docker container, or in the cloud with Azure or AWS. Endpoints can be run on [a variety of platforms](/nservicebus/upgrades/supported-platforms.md).  
```